src/System.Drawing.Primitives/tests/PointFTests.cs

```diff
@@ -118,9 +125,29 @@ public void EqualityTest(float x, float y)
         }
 
         [Fact]
-        public void ToStringTest()
+        public static void EqualityTest_NotPointF()
+        {
+            var point = new PointF(0, 0);
+            Assert.False(point.Equals(null));
+            Assert.False(point.Equals(0));
+            Assert.False(point.Equals(new Point(0, 0)));
+        }
+
+        [Fact]
+        public static void GetHashCodeTest()
         {
-            PointF p = new PointF(0, 0);
+            var point = new PointF(10, 10);
+            Assert.Equal(point.GetHashCode(), new PointF(10, 10).GetHashCode());
+            Assert.NotEqual(point.GetHashCode(), new PointF(20, 10).GetHashCode());
+            Assert.NotEqual(point.GetHashCode(), new PointF(10, 20).GetHashCode());
```

Comment:
I think a comment would suffice. If someone changes `GetHashCode()` and it fails on this case then it would be suspicious in terms of distribution at least and should be investigated in that regard. If it just happens to coincidentally collide on this particular value but have good overall distribution, then they can change the test and explain why suspicions should be allayed in the course of the PR.